src/buffer/out/textBuffer.hpp

```diff
@@ -162,7 +162,10 @@ class TextBuffer final
                               const std::wstring_view fontFaceName,
                               const COLORREF backgroundColor);
 
-    static HRESULT Reflow(TextBuffer& oldBuffer, TextBuffer& newBuffer);
+    static HRESULT Reflow(TextBuffer& oldBuffer,
+                          TextBuffer& newBuffer,
+                          const std::optional<Microsoft::Console::Types::Viewport> lastCharacterViewport = std::nullopt,
+                          short* const lastScrollbackRow = nullptr);

```

Comment:
Just to confirm, even though our practice is generally to move to `std::optional<short>`, you chose a pointer here because...?

Reply:
Well this one is an optional _out param_

Reply:
I had the same thought, but then I read the code :wink:

Reply:
It needed to be in/out, I wasn't sure that an `optional<>&` would work like that but what was I thinking of course that works like that